src/main/resources/static/general.css

```diff
@@ -1,14 +1,23 @@
 #page-container {
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
 }
 
 #content-wrap {
-  flex: 1;
+    flex: 1;
 }
 
 #footer {
-bottom: 0;
-width: 100%;
-}
\ No newline at end of file
+    bottom: 0;
+    width: 100%;
+}
+
+:is(html[lang=en], html[lang=fr]) * {

```

Comment:
Since this seems to be hard coded can you add the step on what to set here to 
https://github.com/Frooodle/Stirling-PDF/blob/main/HowToAddNewLanguage.md

Reply:
I could add the language direction (and their name in the native script) to the properties file. Then dynamically populate them in the template. It might be easier than making changes in multiple places